downstream/modules/platform/proc-controller-add-users-job-templates.adoc

```diff
@@ -4,13 +4,20 @@
 
 .Procedure
 
-. From the navigation panel, select {MenuAMCredentials}.
+. From the navigation panel, select {MenuAECredentials}.
 . Select the credential that you want to assign to additional users.
-. Click the *Access* tab.
+. Click the *User Access* tab.
 You can see users and teams associated with this credential and their roles.
-. Choose a user and click btn:[Add].
 If no users exist, add them from the *Users* menu.
-For more information, see xref:assembly-controller-users[Users].
-. Select *Job Templates* to display the job templates associated with this credential, and which jobs have run recently by using this credential.
+For more information, see TBD[Users].
+. Select the user(s) that you want to give access to the credential and click btn:[Next].
+. From the *Select roles to apply* page, select the roles you want to add to the User.
+. Click btn:[Next].
```

Comment:
I think you have to click Finish here to get to the Add roles dialog that shows success or failure and you click Close within that dialog.

Reply:
So, 
. From the *Select roles to apply* page, select the roles you want to add to the User.
. Click btn:[Next].
. Review your selections and click btn:[Finish] to add the roles or click btn:[Back] to make changes.
+
The Add roles window displays stating whether the action was successful.
+
. Click btn:[Close]. 
+
The *User Access* page displays with the summary information.
+
...